contracts/test/contract.SL2RD.test.js

```diff
@@ -109,7 +109,9 @@ contract("SL2RD", (accounts) => {
       );
 
       // Increment the current count of community slots.
-      await splitContract.transferNextAvailable(accounts[i]);
+      await splitContract.transferNextAvailable(
```

Comment:
@DeusExM4ch1n4 Why a static address here rather than iterating over accounts?

Reply:
The original "i" was a typo. We just want to see if the counter actually works and keeps track of the increment as a helper function. We know from other tests that the logic of the transferNext itself is proper.